lib/service-manager.js

```diff
@@ -92,14 +92,20 @@ prototype.initOrUpdateDb = function(meshApp, callback) {
 
   function stopStaleProcesses(err) {
     if (err) return callback(err);
-    Process.find({where: {stopReason: ''}}, function(err, procs) {
+    Process.find(function(err, procs) {
       if (err) return callback(err);
       async.each(procs, stopProcess, callback);
     });
   }
 
   function stopProcess(proc, callback) {
-    debug('mark stopped: pid %s wid %d', proc.pid, proc.workerId);
+    // Anything other than null, undefined, or '' is a reason.
+    if (proc.stopReason == null || proc.stopReason === '') {
+      console.log('NOT mark stopped: pid %j wid %j reason %j',
+            proc.pid, proc.workerId, proc.stopReason);
+      return callback();
```

Comment:
I think the conditional is inverted.. `null` or `''` are the cases that _should_ be getting marked as stopped, aren't they?